masquerade-core/src/main/java/com/flipkart/masquerade/processor/SerializationOverrideProcessor.java

```diff
@@ -117,19 +117,17 @@ protected boolean skipAnnotationProcessing(FieldMeta field) {
     @Override
     protected void handleFieldKeys(Class<?> clazz, FieldMeta field, MethodSpec.Builder methodBuilder) {
         resolveInclusionLevel(clazz, field);
-        if (field.getInclusionLevel() != JsonInclude.Include.ALWAYS) {
```

Comment:
inclusion level?

Reply:
This change was done by Vishwa (and has been running on prod). I don't have context on this.